linters/groovy/.groovylintrc.json

```diff
@@ -2,13 +2,19 @@
   "extends": "recommended",
 
   "rules": {
+    "convention.CompileStatic": "off",
```

Comment:
what does this do?

Reply:
It disable this warning in the jenkinsfile

```
jenkins/catapult/jenkins/catapult-client-build-base-image-all.groovy
  1     warning  Class should be marked with one of @GrailsCompileStatic, @CompileStatic or @CompileDynamic  CompileStatic 
  ```